scripts/vscode-run-tests.ps1

```diff
@@ -1,14 +1,19 @@
 param (
   [Parameter(Mandatory = $true)][string]$filePath,
-  [Parameter(Mandatory = $false)][string]$framework,
-  [Parameter(Mandatory = $false)][string]$filter
+  [string]$msbuildEngine = "vs",
+  [string]$framework = $null,
+  [string]$filter = $null
 )
 
 Set-StrictMode -version 3.0
 $ErrorActionPreference = "Stop"
 
 . (Join-Path $PSScriptRoot "../eng/build-utils.ps1")
 
+# Run a build
+. (Join-Path $PSScriptRoot "./vscode-build.ps1") -filePath $filePath -framework $framework -msbuildEngine $msbuildEngine
+Write-Output ""
```

Comment:
Why are you using `Write-Output` here instead of `Write-Host`?

Reply:
Write-Host appears to be discouraged as a matter of practice.

https://stackoverflow.com/a/54126870/2855742
https://github.com/PowerShell/PSScriptAnalyzer/blob/361dbf5caf3848e381a999570d971633ca40b818/RuleDocumentation/AvoidUsingWriteHost.md

Reply:
One of the cases it's encouraged for is when the intent is to display content on the screen. That is generally what we're doing here which is why I asked. 

It's not a huge deal for our infra though. The difference is really minimal except in a couple of corner cases